src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -22,12 +27,16 @@ public class ViewLoadedParams : ReadyParams
         private readonly DynamoViewModel dynamoViewModel;
         public readonly Menu dynamoMenu;
 
+        public IWatch3DView BackgroundPreView { get { return dynamoView.BackgroundPreview; } }
```

Comment:
Exposing `Watch3DView` as an `IWatch3DView` property in `ViewLoadedParams` class so that it can be passed to `DMExtension`.